changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 10.3.0 - xxxx-xx-xx =
+* Dev - Requires milestones to be set on pull requests
```

Comment:
Nit: I noticed this while merging changes from `develop`. Can we use `Require` instead of `Requires` so we have a slightly more active voice in the changelog entries?